localflavor/ch/forms.py

```diff
@@ -7,19 +7,20 @@
 import re
 
 from django.core.validators import EMPTY_VALUES
+from django import forms
```

Comment:
Indeed it is not, I can remove it. I did it to follow other Field implementations :)